Cards/MadGraph5_aMCatNLO/DY/DYto2L-4Jets_MLL-10to50_madgraphMLM-pythia8/DYto2L-4Jets_MLL-10to50_madgraphMLM-pythia8.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
A bit wide scope question, not about the card itself. Do we really need DY MLM 10to50? I am wondering who might be the users for this. Maybe some SUS with lepton jet multiple final states?

Reply:
Otherwise, cards look fine

Reply:
Right. Usually the stats from this is super low.